WriterDesk1/backend/.dockerignore

```diff
@@ -0,0 +1,23 @@
+**/.classpath
```

Comment:
Horen er meerdere .dockerignore files te zijn?

Reply:
Vgm wel, is automatisch gegenereerd